apps/hermes/client/js/src/index.ts

```diff
@@ -0,0 +1,10 @@
+export * from "./types";
+
+export {
+  DurationInMs,
+  DurationInSeconds,
+  HermesConnection,
+  HermesConnectionConfig,
+  HexString,
+  UnixTimestamp,
+} from "./HermesConnection";
```

Comment:
I have generally moved away from using barrel files.  The reason is that there are many environments where barrel files don't tree-shake correctly ([nextjs itself had this problem for many years](https://github.com/vercel/next.js/issues/12557) and may still have it, I'm honestly not sure and the thread about it is extremely long and split across many discussions and tickets so it's hard to say without just testing it) and it adds overhead to the bundler and require resolution that node doesn't compile away very effectively.

It obviously doesn't matter much for a case like this where it's a single layer of indirection and there's nothing that _could_ be shaken anyways, but patterns get copied and libraries get extended over the years, so I advise just avoiding the pattern to start with.

[Package exports are your friend](https://nodejs.org/api/packages.html#exports)